docs/webhook/examples/payment-payload.mdx

```diff
@@ -40,6 +40,11 @@ Os webhooks de `Pagamento Externo com Falha (Pix Out)` enviam objetos com os seg
 
 - `payment`: Objeto com detalhes do Pagamento
 - `transaction`: Objeto com detalhes da Transação Pix se ela existir
+- `error`:  Objeto com o código e descrição do motivo da falha.
```

Comment:
Can you describe the object?

Reply:
it is describe in the example linked with the documentation on info section